code/controllers/subsystem/radioactive_nebula.dm

```diff
@@ -0,0 +1,56 @@
+#define TRAIT_RADIOACTIVE_NEBULA_FAKE_IRRADIATED "radioactive_nebula_fake_irradiated"
+
+/// Controls making objects irradiated when Radioactive Nebula is in effect.
+SUBSYSTEM_DEF(radioactive_nebula)
+	name = "Radioactive Nebula"
+	flags = SS_BACKGROUND
+	wait = 30 SECONDS
+
+	VAR_PRIVATE
+		datum/station_trait/nebula/hostile/radiation/radioactive_nebula
+
+/datum/controller/subsystem/radioactive_nebula/Initialize()
+	radioactive_nebula = locate() in SSstation.station_traits
+	if (!radioactive_nebula)
+		can_fire = FALSE
+		return SS_INIT_NO_NEED
+
+	// We don't *really* care that this happens by the time the server is ready to play.
+	ASYNC
+		irradiate_everything()
+
+	return SS_INIT_SUCCESS
```

Comment:
Will this tell everyone that the trait is active by watching the subsystems initialize?

Reply:
Yes it will, it should just be done in SSmapping or SSstation

Reply:
It should definitely not be done on mapping or station because this consumer does not control the processing of those subsystems. It is also reasonable for us to want subsystems that can initialize without exposing meta information (like Reebe or whatever).

Will add a new `SS_INIT_` return value that hides the message (but is named something other than NO_NEED)